org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/XMLDocumentSymbolsTest.java

```diff
@@ -143,4 +143,21 @@ public void exceedSymbolLimit() {
 		settings.setMaxItemsComputed(9);
 		testDocumentSymbolsFor(xml, "test.xml", settings, symbol1, symbol2);
 	}
+
+	@Test
+	public void limitOfZero() {
+		String xml = "<a>\n" + //
+				"  <b>\n" + //
+				"    <c>\n" + //
+				"      <d />\n" + //
+				"    </c>\n" + //
+				"  </b>" + //
+				"</a>";
+
+		XMLSymbolSettings settings = new XMLSymbolSettings();
+		settings.setMaxItemsComputed(0);
+
+		testDocumentSymbolsFor(xml, "test.xml", settings);

```

Comment:
what is the assertion here?

Reply:
These are no document symbols when the limit is zero. I will make this test a bit more clear.

Reply:
Please let me know if this clarifies the test